elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchJson.java

```diff
@@ -97,42 +95,33 @@ static void visitMappingProperties(ObjectNode mapping,
       BiConsumer<String, String> consumer) {
     Objects.requireNonNull(mapping, "mapping");
     Objects.requireNonNull(consumer, "consumer");
-    visitMappingProperties(new ArrayDeque<>(), mapping, consumer);
+    visitMappingProperties("", mapping, consumer);
   }
 
-  private static void visitMappingProperties(Deque<String> path,
-      ObjectNode mapping, BiConsumer<String, String> consumer) {
+  private static void visitMappingProperties(String prefix, ObjectNode mapping,
+                                             BiConsumer<String, String> consumer) {
     Objects.requireNonNull(mapping, "mapping");
```

Comment:
Please fix all these indents.

Reply:
> Please fix all these indents.

It's automatically indents by IDEA when I break line, and meets the default code check style.

Reply:
Let's follow other Calcite code's continuous indent, the default code check didn't check this but we should follow the common pattern.

Reply:
@danny0405 thanks your advice, I have fix these extra indents in my new commit.